commute:0.1.0

giacomogallina opened the pull request:
<!--
Thanks for submitting a package! Please read and follow the submission guidelines detailed in the repository's README and check the boxes below. If you want to make a PR for something other than a package submission, just delete all this and make a plain PR.
-->

I am submitting
- [x] a new package
- [ ] an update for a package

I have read and followed the submission guidelines and, in particular, I
- [x] selected a name in conformance with the guidelines
- [x] added a `typst.toml` file with all required keys
- [x] added a `README.md` with documentation for my package
- [x] have chosen a license and added a `LICENSE` file or linked one in my `README.md`
- [x] ensured that my submission does not infringe upon the rights of a third party
- [x] tested my package locally on my system and it worked
- [x] named this PR as `name:version` of the submitted package
- [x] agree that my package will not be removed without good reason

<!--
Please add a brief description of your package below and explain why you think it is useful to others.
-->

Description: This package can create commutative diagrams, useful for many fields of mathematics. It has a decent amount of features and it's relatively easy to use.

Sorry for any rookie mistake i might have made, i think this is the first time that i do a pull request

laurmaedje commented on packages/preview/commute-0.1.0/README.md:
The more canonical style in Typst would also be kebab-case (i.e. `commutative-diagram`), but that's also up to you.

giacomogallina replied:
thanks for the suggestions, i'll update the pull request later!

giacomogallina commented:
Hi, i updated the pull request with a new commit where i switch everything to kebab-case (it's a bit weird, but i guess i'll get used to it). I also moved the code example to the top of the readme, it was a nice suggestion.

laurmaedje commented:
Thank you!
